src/@next/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,268 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Portal } from '../Portal/Portal';
+import { Typography } from '../Typography';
+import {
+  bottomPositionResolver,
+  leftPositionResolver,
+  rightPositionResolver,
+  topPositionResolver,
+} from './positionResolver';
+import {
+  canAlignBottom,
+  canAlignCenter,
+  canAlignLeft,
+  canAlignMiddle,
+  canAlignRight,
+  canAlignTop,
+  horizontalBuffer,
+  verticalBuffer,
+} from './positionResolverHelper';
+import {
+  StyledTooltip,
+  StyledTooltipArrow,
+  StyledTooltipContainer,
+} from './TooltipStyle';
+const topPositions = ['top-center', 'top-left', 'top-right'] as const;
```

Comment:
add a new line; it's kinda hard to read without separating

Reply:
done